sdk/src/azure/storage/az_storage_blobs_blob_client.c

```diff
@@ -163,7 +162,7 @@ AZ_NODISCARD az_result az_storage_blobs_blob_upload(
   az_http_request request;
   AZ_RETURN_IF_FAILED(az_http_request_init(
       &request,
-      context,
+      opt.context,
```

Comment:
We no longer need the `(void)` on line 147 now:
https://github.com/RickWinter/azure-sdk-for-c/blob/0cd5306e9193dcdd133ef0a2ce102d045d7526c9/sdk/src/azure/storage/az_storage_blobs_blob_client.c#L147